astrocut/cube_cut.py

```diff
@@ -844,8 +843,9 @@ def cube_cut(
             if unsuccessful returns None.
         """
 
-        if verbose:
-            start_time = time()
+        start_time = time()
```

Comment:
not a huge deal, but we should be using `time.monotonic()` or `time.perf_counter()` here instead.

https://docs.python.org/3/library/time.html#time.monotonic